### PR TITLE
Feat Added Add To Cart. Add To Wishlist Functionalities

### PR DESCRIPTION
**Pull request by Asuraking1n**
1-Cart Working (Total price not calculating yet)

2-Wishlist Working

3-New user Can Signup
4-User Can Logout

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -37,10 +37,13 @@
 }
 
 *::-webkit-scrollbar-thumb {
-  background-color: #a7ece6;
+  background-color: #a7ece6;
```
Comment (Divya70) [inline src/App.css:40]: use variable name for color

File: src/components/cards/priceCard/PriceCard.js [hunk 2]
```diff
@@ -1,34 +1,93 @@
...
-        </div>
-        
-    </>
-  )
-}
+        </>
+    );
+};
 
-export default RatedCard
```
Comment (Divya70) [inline src/components/cards/priceCard/PriceCard.js:34]: add an extra blank line to not get such warnings from github

**Review by Divya70 — APPROVED** [review]
suggested some chnages else everything look fine

File: src/App.css (hunk 1 above)
Comment (Asuraking1n) [reply src/App.css:40]: i will added that later in last

**Pull request merged**